### ISSUE TITLE
All words spelled incorrectly, no suggestions

### ISSUE THREAD

**Issue by treb0r** [bartosz-antosik/vscode-spellright#264]
I am trying to configure Spell Right for VS Code on Ubuntu.

I have followed the install instructions and downloaded the dictionaries.

I see the status bar asking me the default language, and I can select English (British).

All words are underlined in red, no suggestions showing.

I've tried reinstalling, restarting VS Code and replacing the dictionaries.

What am I doing wrong?

**Comment by bartosz-antosik** [comment]
Hi! It is hard to tell. I have it running on Ubuntu (Kubuntu to be precise). Maybe it is some security mechanism - could you have a look in the logs?

![kubuntu](https://user-images.githubusercontent.com/20492548/55354964-4a1ab480-54c7-11e9-9c66-33edeacc4b0a.png)

**Comment by treb0r** [comment]
Thanks for the reply!

Nothing in the logs. 

I'll keep looking for a solution and will post it here if I find one.

**Comment by bartosz-antosik** [comment]
I have been thinking about the situation for some time and I could pose few questions if only you are still interested in using the extension (I would perfectly understand if you would switched to any other if it would just be working for you).

You can switch the dictionaries in the status (if you have more than one) bar and thir names are read correctly?

You have verified that the dictionaries look ok inside (both dic and aff files) and they are UTF? Have you downloaded the dictionaries from the source provided in the README file?

Is there maybe a chance you could test this on another machine? Just to see if this is a regular or incidental error (possibly bound to particular configuration etc.)

I have tested this here and there on Virtual Machines and it seems to be working ok.

**Comment by joelhans** [comment]
I was having this exact same issue and I just found a solution that worked for me.

When I downloaded the `.aff` and `.dic` files from [this link](https://github.com/titoBouzout/Dictionaries) in the documentation, they were named `English (American).aff` and `English (American).dic`, respectively. I wondered if maybe the space and/or parentheses could be causing an issue.

I renamed the files `English.aff` and `English.dic` and the issue disappeared right away. I tested a bit and it *seems* like the parentheses are the issue. Everything works as expected now. I hope this helps solve your issue, too!

I'm on Fedora 29. Used the dictionary files from that GitHub repo linked above because those that come with `hunspell` on my system are `ISO8859-1`, not `UTF`.
